windows/9.x.x/sitecore-xp-sqldev/Boot.ps1

```diff
@@ -44,8 +44,8 @@ Write-Host "$(Get-Date -Format $timeFormat): Preparing Sitecore databases..."
 # See http://jonnekats.nl/2017/sql-connection-issue-xconnect/ for details...
 Invoke-Sqlcmd -Query ("EXEC sp_MSforeachdb 'IF charindex(''Sitecore'', ''?'' ) = 1 BEGIN EXEC [?]..sp_changedbowner ''sa'' END'")
 Invoke-Sqlcmd -Query ("UPDATE [Sitecore.Xdb.Collection.ShardMapManager].[__ShardManagement].[ShardsGlobal] SET ServerName = '{0}'" -f $SqlHostname)
```

Comment:
Something is wrong here, the prefix is not used in the formatting string in the 3 UPDATE statements...